src/spaceone/inventory_v2/manager/namespace_group_manager.py

```diff
@@ -30,8 +30,10 @@ def _rollback(vo: NamespaceGroup):
             )
             vo.delete()
 
-        if "namespace_group_id" not in params:
+        namespace_group_id = params.get("namespace_group_id")
+        if namespace_group_id is None:
```

Comment:
```suggestion
        if namespace_group_id := params.get("namespace_group_id"):
```

Reply:
This is not a mandatory change. It's a code example aimed at reducing the number of lines.

Reply:
ok changed
```suggestion
        if namespace_group_id := params.get("namespace_group_id"): 
            params["namespace_group_id"] = utils.generate_id("nsg")
```